middle_end/flambda/unboxing/is_unboxing_beneficial.ml

```diff
@@ -93,3 +93,9 @@ let rec filter_non_beneficial_decisions decision : U.decision =
     as decision ->
     if is_unboxing_beneficial_for_epa epa then decision
     else Do_not_unbox Not_beneficial
+
+let filter_non_beneficial_decisions ~continuation_is_recursive decision =
+  if continuation_is_recursive then decision
+  else filter_non_beneficial_decisions_aux decision
```

Comment:
I think that it would be clearer to move that if to the call site rather than here, and leave that file unchanged. Or maybe rename to filter_non_beneficial_decision_for_non_recursive_continuation or something like that

Reply:
done